MobileWallet/Screens/Send/AddRecipient/AddRecipientViewController.swift

```diff
@@ -163,16 +163,19 @@ class AddRecipientViewController: UIViewController, UITextFieldDelegate, Contact
 
     override func viewDidAppear(_ animated: Bool) {
         super.viewDidAppear(animated)
-
         if selectedRecipientPublicKey == nil {
             isEditingSearchBox = true
         }
+        if selectedContact != nil {
+            selectedContact = nil
+            selectedRecipientPublicKey = nil
+            inputBox.text = nil
```

Comment:
This is the desired behavior for all cases - the screen should be initialized when the user comes back to it. @SerhiiShovkoplias @Jasonvdb

Reply:
I think if we pasted non-contact emojiID we should keep it into textField when back to this controller. Into another cases text field should be clean. @kukabi